.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
But this is project based configurations. The `node_modules/` directory is belonging to the project itself.
The global root .gitignore should have auto-generated system files, `*.DS_Store` for example or compiled files like (`.exe`, or `/build` and `/dist` directories)
You may need to keep it tracked in another project.